### PR TITLE
feat(ui): improve requirements modal with models type

### LINKED ISSUE #1503: fix(api): don't return models if you don't have right to use them

### LINKED ISSUE #1504: feat (ui): add icon for official worker model (in list select)
if group == "shared.infra" --> official
else non official

### PR DESCRIPTION
**Pull request by bnjjj**
close #1503 
close #1504

### CODE REVIEW

File: ui/src/assets/i18n/fr.json [hunk 1]
```diff
@@ -258,7 +258,7 @@
   "navbar_groups" : "Groupes",
   "navbar_home" : "Accueil",
   "navbar_profile" : "Votre Compte",
-  "navbar_projects_placeholder": "Tous les projects",  
+  "navbar_projects_placeholder": "Tous les projects",
```
Comment (fsamin) [inline ui/src/assets/i18n/fr.json]: projets

File: ui/src/app/shared/requirements/form/requirements.form.component.ts [hunk 2]
```diff
@@ -68,6 +66,30 @@ export class RequirementsFormComponent {
         }
     }
 
+    ngOnInit() {
+        this._workerModelService.getWorkerModels().first()
```
Comment (sguiheux) [inline ui/src/app/shared/requirements/form/requirements.form.component.ts]: getWorkerModels().pipe(first(), finalize(...      possible with the current version of rxjs ?

**Comment by yesnault** [comment]
rebase master done (to run UT)

**Comment by bnjjj** [comment]
up to date

**Pull request merged**